third_party/2and3/cryptography/exceptions.pyi

```diff
@@ -0,0 +1,2 @@
+class InvalidSignature(Exception): ...
```

Comment:
Please add `__getattr__(name: str) -> Any: ...  # incomplete` to the top of this and other files that have more definitions than the stubs.